index.html

```diff
@@ -28,14 +37,14 @@ <h1>about me:</h1>
     world for much of my life and so I have a deep need to feel like my presence and 
     existence impact it in some tangible way. The most fulfilling way I've found to achieve
     that is by making something. Taking something from within myself and seeing it manifest
-    outside of myself. Making it real, and therefore, making myself real. If the fabric 
+    outside of myself. Making it real"Times New Roman", , and therefore, making myself real. If the fabric 
```

Comment:
Is this perhaps a typo that was accidentally added when you were changing fonts?